web/skins/classic/views/js/montagereview.js.php

```diff
@@ -118,13 +118,13 @@
   echo "var maxScore=$maxScore;\n";  // used to skip frame load if we find no alarms.
 } // end if initialmodeislive
 
-echo "var Storage = [];\n";
+echo "\nvar Storage = [];\n";
```

Comment:
The \n at the beginning there looks a bit odd. Just checking this is by design.

Reply:
Mostly there for readability of resulting javascript. separates logical sections with a new line.